src/test/scala-2.11/markets/participants/MarketParticipantSpec.scala

```diff
@@ -27,7 +27,7 @@ import markets.orders.market.MarketBidOrder
 import markets.tradables.{Tradable, TestTradable}
 import org.scalatest.{FeatureSpecLike, GivenWhenThen, Matchers}
 
-import scala.collection.{immutable, mutable}
+import scala.collection.mutable
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2168327547/issues/source?bid=2921361&fileBranchId=3162043#l30)